frontend/src/components/school/SchoolDetails.vue

```diff
@@ -0,0 +1,501 @@
+                        </v-row>
+                        <v-row>
+                          <span>{{address.postal}}</span>
+                        </v-row>
+                      </v-col>
+                    </v-row>
+                  </v-col>
+                </v-row>
+              </v-col>
+              <v-col cols="4" lg="3" class="pb-0 pt-0">
+                <v-row class="mb-n4">
+                  <v-col cols="10" class="pb-2 pt-2 pr-0">
+                    <v-icon aria-hidden="false" class="pr-3">
+                      mdi-home-outline
+                    </v-icon>
+                    <span style="color: grey">Physical Addresses</span>
+                  </v-col>
+                </v-row>
+                <v-row>
+                  <v-col cols="10" class="pb-1 pr-0" v-for="address in school.addresses" :key="address.addressId">
+                    <v-row v-if="address.addressTypeCode === 'PHYSICAL'">
+                      <v-col>
+                        <v-row>
+                          <span class="ministryLine" style="color: black">{{ address.addressLine1 }}</span>
+                        </v-row>
+                        <v-row>
+                          <span>{{ address.city }}, {{ address.provinceCode }} {{getCountryName(address.countryCode)}}</span>
+                        </v-row>
+                        <v-row>
+                          <span>{{address.postal}}</span>
+                        </v-row>
+                      </v-col>
+                    </v-row>
+                    <v-row class="pl-4" v-else>
+                      <span>Same as Mailing Address</span>
+                    </v-row>
+                  </v-col>
+                </v-row>
+              </v-col>
+            </v-row>
+          </div>
+        </v-col>
+      </v-row>
+    </div>
+  </v-container>
+</template>
+
+<script>
+
+import PrimaryButton from '../util/PrimaryButton';
+import {mapGetters} from 'vuex';
+import alertMixin from '@/mixins/alertMixin';
+import ApiService from '@/common/apiService';
+import {ApiRoutes} from '@/utils/constants';
+
+export default {
+  name: 'SchoolDetailsPage',
+  mixins: [alertMixin],
+  components: {
+    PrimaryButton,
+  },
+  data() {
+    return {
+      school: '',
+      district: '',
+    };
+  },
+  computed: {
+    ...mapGetters('auth', ['isAuthenticated','userInfo']),
+    dataReady: function () {
+      return this.userInfo;
+    }
+  },
+  created() {
+    this.getThisSchoolsDetails();
+  },
+  methods: {
+    
+    getThisSchoolsDetails(){
+      this.school = '';
+
+      ApiService.apiAxios.get(ApiRoutes.school.SCHOOL_DETAILS_BY_ID + `/${this.userInfo.activeInstituteIdentifier}`)
+        .then(response => {
+          this.school = response.data;
+          this.populateExtraSchoolFields(this.school);
+          this.getDistrictDetails(this.school.districtId);
+        }).catch(error => {
+        //to do add the alert framework for error or success
```

Comment:
Ahh there's an existing alert framework for this. https://github.com/bcgov/EDUC-EDX/blob/master/frontend/src/mixins/alertMixin.js 

take a peek at this for implementation https://github.com/bcgov/EDUC-EDX/blob/master/frontend/src/components/SecureExchange/MessageDisplay.vue#:~:text=import%20alertMixin%20from%20%27%40/mixins/alertMixin%27%3B